fluentd/parser_viaq_host_audit.rb

```diff
@@ -0,0 +1,27 @@
+require 'fluent/parser'
```

Comment:
You could move this to the fluentd/lib dir and add tests... package it like a proper gem..  we have unit tests that run all the tests in  the lib dir

Reply:
ack; will do.